.eslintrc

```diff
@@ -1,21 +1,33 @@
 {
   "env": {
-		"ecmaVersion": 2020,
-		"browser": true,
-		"mocha": true,
-		"node": true
+    "es6": true,
```

Comment:
`ecmaVersion` seems to be a valid property, according to ESLint doc, but I think it's only available on newer eslint version (v7), while ours eslint is still on v5.